test/integration/test_options.py

```diff
@@ -98,3 +99,28 @@ def test_simulator_transpile(self, service):
                         job2.result()
                     # TODO: Re-enable when ntc-1651 is fixed
                     # self.assertIn("TranspilerError", err.exception.message)
+
+    @run_integration_test
+    def test_optimization_level(self, service):
+        """Test various definitions for optimization_level."""
+
+        backend = service.backends(simulator=True)[0]
```

Comment:
nit: ```service.get_backend('ibmq_qasm_simulator")```

Reply:
No problem, but this line appears in several tests in this file. Shall I change them all to be consistent?

Reply:
ah I didn't notice that - you can leave it as is then